chapters/background.tex

```diff
@@ -1,4 +1,12 @@
 \section{Background} \label{sec:background}
 % identifying existing approaches within the research literature; this may potentially include mobile-based and non-mobile approaches if necessary
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.
-
+The renaissance of machine learning in the 2010s outclassed this approach because it lacks the ability to detect emotions/smiles.
+Regarding soHappy, it led to TensorFlow becoming a more considerable option for emotion detection.
+TensorFlow is a open source machine learning library developed by Google.
+It supports a wide variety of programming languages, mainly targeting web and mobile applications.
+For mobile development TensorFlow provides a Lite version called TensorFlow Lite (TFLite) which can be easily included in a mobile app.
+The main advantage TFLite offers, is the ability to add a self-trained model to detect smiles. This is very helpful to reduce its more advanced emotion detection features to a simpler smile detection model.
+As a second option we added a Face/Smile Detector implemented in Google's Android API (ML Kit) which also uses TensorFlow.
+ML Kit comes with a model trained using deep machine learning (DML).
```

Comment:
Isn't it usually referred to as "deep learning" instead of "deep machine learning"?